snapcraft/application.py

```diff
@@ -180,7 +180,7 @@ def _run_inner(self) -> int:
                         f"{store.constants.ENVIRONMENT_STORE_CREDENTIALS} "
                         "is correctly exported into the environment"
                     ),
-                    docs_url="https://snapcraft.io/docs/snapcraft-authentication",
+                    docs_url="https://documentation.ubuntu.com/snapcraft/stable/how-to/publishing/authenticate",
```

Comment:
Unrelated: I didn't realize we had so many errors still setting `docs_url`.  At some point, we should replace `docs_url` for `docs_slug`, since we set the base url [here](https://github.com/canonical/snapcraft/blob/9d7a82a196ead5fd98164bd28cf7bdd9aecafa2e/snapcraft/application.py#L53).  That ensures the version is correct in the url.

Reply:
I'm seeing a mismatch between what's failing locally in the fast test and what the remote test says. The only errors I get locally are patchelf issues in the tests. Is the URL change having other side effects?

Reply:
Tests should (:crossed_fingers:) be passing now. https://github.com/canonical/snapcraft/pull/5528/commits/6497cec26285576bcd2deae1c82d3ffdccd1c32b

Thanks for the help!